scripts/monitoring/cron-send-zabbix-inventory-check.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+''' collect info of zabbix and compare with info on master  '''
+
+# TODO: change to use arguments for podname, and run once per podname
+
+#pylint: disable=import-error
+#pylint: disable=invalid-name
+#pylint: disable=broad-except
+
+import argparse
+import logging
+import json
+import urllib2
+import sys
+sys.path.insert(0, '/container_setup')
+
+from urllib2 import Request, urlopen, URLError, HTTPError
+
+from openshift_tools.monitoring.ocutil import OCUtil
+from openshift_tools.monitoring.metric_sender import MetricSender
+from zabbix_hostinfo import node_number_onzabbix
+
+logging.basicConfig(
+    format='%(asctime)s - %(relativeCreated)6d - %(levelname)-8s - %(message)s',
+)
+logging.getLogger().setLevel(logging.WARN)
+
+class ZabbixInfo(object):
+    '''
+      this will check the zabbix data and compare it with the real world
+    '''
+    def __init__(self, args=None, ):
+        '''initial for the InfraNodePodStatus'''
+        self.args = args
+        self.kubeconfig = '/tmp/admin.kubeconfig'
+        self.oc = OCUtil(namespace=self.args.namespace, config_file=self.kubeconfig)
+
+    def get_all_hosts(self):
+        ''' get all the hosts  '''
+        pods = self.oc.get_nodes()
+        logging.getLogger().info("number from master is :"+str(len(pods['items'])))
```

Comment:
better messaging:
"oc get nodes returns {count} hosts"
"zabbix shows {count} hosts"

Reply:
can we also have "host {abc} not in zabbix"? maybe "host {abc} in zabbix but not in cluster"?